criu/timerfd.c

```diff
@@ -75,6 +75,12 @@ static int dump_one_timerfd(int lfd, u32 id, const struct fd_parms *p)
 	tfe.id = id;
 	tfe.flags = p->flags;
 	tfe.fown = (FownEntry *)&p->fown;
+
+	/* when it_value is zero, we need parse it again */
```

Comment:
Could you write a more details comment here?  Right now, it is unclear why we need to parse it again.